requirements.txt

```diff
@@ -4,7 +4,7 @@ future>=0.16.0
 h5py>=2.5.0
 pylibtiff>=0.4.2
 matplotlib>=1.4.3,<3.0
-numpy>=1.11.0,<1.18
+numpy>=1.11.0,<1.26

```

Comment:
This file is not really magical. It's not going to get a different numpy version installed on the testing computer just by upping the version number. We haven't tested Odemis with numpy > 1.18, so I'm a little hesitant. Note that Ubuntu 22.04 ships with numpy 1.21, so we should test with such version soon, in any case!

It's actually possible to install numpy 1.20 without changing this file. So I'd suggest to just fix the installation document only.